gui/wxpython/datacatalog/tree.py

```diff
@@ -981,17 +982,25 @@ def OnDoubleClick(self, node):
                     dlg = wx.MessageDialog(
                         parent=self,
                         message=_(
-                            "Map <{0}@{1}> is not in the current location"
-                            " and therefore cannot be displayed."
-                            "\n\n"
-                            "To display this map switch to mapset <{1}> first."
+                            "Map <{0}@{1}> is not in the current location. "
+                            "Do you want to switch to the location '{2}'? \n\n"
+                            "All current Map Displays will be closed."
```

Comment:
This looks like "will be closed in any case". I think the new paragraph needs to be between "...the current location" and "All current Map...".

Reply:
Ok, then I would suggest this formulation:
![easier_switch](https://user-images.githubusercontent.com/49241681/120173687-90cddb80-c204-11eb-9ab7-c1843b31b522.PNG)

Reply:
That message sounds a bit intimidating to me... Can we say something like: 

```
Map <bla> is not in the current location. To be able to display it you need to switch to <blabla> location. Note that if you switch there all current Map Displays will be closed. 

Do you want to switch anyway?
```

Too long? My intention is to "explain" in short that to display a map from a different location the user needs to switch there.

Reply:
I like your idea more Vero. It is true that it is quite long but we need to fully explain it. @petrasovaa do you agree with that?

Reply:
> That message sounds a bit intimidating to me... Can we say something like:
> 
> ```
> Map <bla> is not in the current location. To be able to display it you need to switch to <blabla> location. 
> Note that if you switch there all current Map Displays will be closed. 
> 
> Do you want to switch anyway?
> ```
> 
> Too long? My intention is to "explain" in short that to display a map from a different location the user needs to switch there.

I think @veroandreo 's formulation is a bit more appealing. However, I think the button labels are too wordy. What about `Cancel` (as default) and `Switch`?

Reply:
![switch](https://user-images.githubusercontent.com/49241681/120225070-e033fc00-c244-11eb-9259-8736b3b433a2.PNG)